src/main/java/coms/w4156/moviewishlist/models/ModelInterface.java

```diff
@@ -0,0 +1,5 @@
+package coms.w4156.moviewishlist.models;
+
+public interface ModelInterface<ID extends Comparable<ID>> {
```

Comment:
All models implement this simple, barebones interface. This is only really used for implementing services using the abstract ServiceForRepository class.